packages/block-editor/src/components/off-canvas-editor/index.js

```diff
@@ -187,29 +187,31 @@ function __ExperimentalOffCanvasEditor(
 				listViewRef={ elementRef }
 				blockDropTarget={ blockDropTarget }
 			/>
-			<TreeGrid
-				id={ id }
-				className="block-editor-list-view-tree"
-				aria-label={ __( 'Block navigation structure' ) }
-				ref={ treeGridRef }
-				onCollapseRow={ collapseRow }
-				onExpandRow={ expandRow }
-				onFocusRow={ focusRow }
-				applicationAriaLabel={ __( 'Block navigation structure' ) }
-			>
-				<ListViewContext.Provider value={ contextValue }>
-					<ListViewBranch
-						blocks={ clientIdsTree }
-						selectBlock={ selectEditorBlock }
-						showBlockMovers={ showBlockMovers }
-						fixedListWindow={ fixedListWindow }
-						selectedClientIds={ selectedClientIds }
-						isExpanded={ isExpanded }
-						shouldShowInnerBlocks={ shouldShowInnerBlocks }
-						selectBlockInCanvas={ selectBlockInCanvas }
-					/>
-				</ListViewContext.Provider>
-			</TreeGrid>
+			<div className="offcanvas-editor-list-view-tree-wrapper">
```

Comment:
This class can't go in `TreeGrid` because it applies to the table element, and you can't overflow tables, you need to target a container or make the table `display; block;`

I was considering being able to pass the class to the wrapper div inside `TreeGrid` (the one that right now is applying the aria-label) but I wasn't sure if that was the right way to go.

Reply:
You may not need this class at all if we avoid the cascade. See other comments in this review.